docs/self-hosted-appcircle/install-server/linux-package/configure-server/container-engine-network-subnet.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Configure Container Engine Network Subnet
+sidebar_label: Configure Container Engine Network Subnet
+description: Configure the Appcircle server to use a specific network subnets for the containers.
+tags: [docker, self-hosted, network]
+sidebar_position: 50
+---
+
+import SpacetechExampleInfo from '@site/docs/self-hosted-appcircle/install-server/linux-package/configure-server/_spacetech-example-info.mdx';  
+import RestartAppcircleServer from '@site/docs/self-hosted-appcircle/install-server/linux-package/configure-server/_restart-appcircle-server.mdx';  
+import DowntimeCaution from '@site/docs/self-hosted-appcircle/install-server/linux-package/configure-server/_appcircle-server-downtime-caution.mdx';  
+
+
+## Overview
+
+
+This document explains how to configure your Appcircle server's network subnets of the containers if the existing subnets are conflicting with other networks or not suitable for the you use case.
+
```

Comment:
Add info box to mention that it's available self-hosted server version `3.28.0` or later.